webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'asset',
-    filename: './asset.js',
+    library: 'AssetService',
```

Comment:
This will be a breaking change to Webpack users. If we're breaking it anyway (which is what is likely to happen), let's keep this. If we want to be compatible, we'll need `--main-service-name asset` as an extra generator arg. But I'd better break it since there are no active Webpack users here as I'm pretty sure :)